### PR TITLE
The IslePathActor's

### PR DESCRIPTION
**Pull request by ItzSwirlz**
And ideally, as much of a basic implementation for all of then. As of opening this PR, I'm too afraid to go into Helicopter and I think another one that has to do with Matrixes, which I'm a bit scared of.. so I'll come back to that. A few classes are also missing IsA's and ClassName's

### CODE REVIEW

**Comment by itsmattkc** [comment]
While you're filling out class structures, you may as well add `DECOMP_SIZE_ASSERT`s to make sure everything is coming out right.

**Comment by ItzSwirlz** [comment]
My bad, I'll do that right now.

I wonder if for the things that are built/have more pieces like the Ambulance, or things that may be built in game, if some fields are certain classes for just plain Bricks - so like when building it, the actor/entity/whatever has one brick together done and not another

File: LEGO1/pizza.cpp [hunk 1]
```diff
@@ -13,7 +13,7 @@ Pizza::Pizza()
   this->m_unk88 = 0;
   this->m_unk8c = -1;
   this->m_unk98 = 0;
-  this->m_unk90 = 0x80000000;
+  this->m_unk90 = 0.0;
```
Comment (itsmattkc) [inline LEGO1/pizza.cpp:16]: I have a feeling this isn't a float. `0.0f` would ordinarily resolve to `0x0` in hex. The `80000000` could be valid for _negative_ `0.0f`, but that would be an extremely unusual thing for a developer to program intentionally. I think leaving this as a U32 is probably more accurate.

**Review by foxtacles — APPROVED** [review]
Fixed the `TowTrack` structure, the rest looks good to me!

**Pull request merged**